### PR TITLE
Answered problem one in super rogue fashion.

### PR DESCRIPTION
**Pull request by erikaasland**

### CODE REVIEW

**Comment by himleyb85** [comment]
Push to a branch instead, I gave you permissions (my bad).  Kudos for figuring out how to branch and open a pull request!  Also, use the existing html and css files in the exercises :)

File: 1-basic-layout/seess.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/* ......*/
+
+header {
+	position: top absolute;
```
Comment (himleyb85) [inline 1-basic-layout/seess.css]: This line isn't doing anything, the syntax is wrong.  If you inspect your page using the dev console you'll see the css that isn't working on the right, it has a little yield sign next to it.  Find the yield sign so you can see what I mean, let me know if you can't.

File: 1-basic-layout/seess.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	position: absolute;
+	background-color: black;
+	text-decoration: none;
+	width:99.15%;
+	text-align: left;
+	font-size: 15px;
+	height: 3%	
+}
+a{
+	text-decoration:blink;
```
Comment (himleyb85) [inline 1-basic-layout/seess.css]: twinky

text a little close to the edge here, try adding `padding-left: .5em` to the `a` and `h1`

File: 1-basic-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+				<div>
+					<a href="https://www.youtube.com/watch?v=31m33PJW7cw">Joe Breezy Challenge</a>
+				</div>
+				<div>
+					<a href="https://www.youtube.com/watch?v=oj0kr1QLjD0">Obesity: America's Greatest Challenge</a>
+				</div>
+			</nav>
+			<section class="content">
+				<br>
+					<table>
```
Comment (himleyb85) [inline 1-basic-layout/index.html]: Don't use tables.  Use divs.
http://stackoverflow.com/a/17503474/3669062

File: 1-basic-layout/index.html [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+   						<tr>
+      						<td><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
```
Comment (himleyb85) [inline 1-basic-layout/index.html]: Indent every nested element.
```
<tr>
    <td>
        <p>
```

**Comment by himleyb85** [comment]
`2em` instead of `1em` is fine, the only other thing I'd change is to make the page responsive (i.e., if viewed on tablet or mobile it still looks nice, see the last line of the requirements)
twinky_small

Otherwise, it's a great start.  Make the other fixes before you push code in the other exercises.

File: 1-basic-layout/seess.css (hunk 1 above)
Comment (erikaasland) [inline 1-basic-layout/seess.css]: Do you me my dev tools in chrome?  If so, I can't find what "!" you are
speaking of. I'll try to solve it regardless.

Comment (himleyb85) [inline 1-basic-layout/seess.css]: Yup.  Right click on the header and `Inspect element`.  Once the console is open, you can select specific elements and dynamically change them (add classes, remove elements on left, change css attributes, add new ones on right).

header

Comment (erikaasland) [inline 1-basic-layout/seess.css]: figured out the the developer tools too.
im done for the day. time to get some fresh air.

**Pull request closed without merging**